components/cockpit.jsx

```diff
@@ -0,0 +1,173 @@
+import styled, {css} from "styled-components";
+import { useState } from "react";
+import { nanoid } from 'nanoid';
+
+export function Cockpit(){
+
+const [commandLine, setCommandLine] = useState([])
+const [count, setCount]= useState(0)
+
+function addRight(){
+if(count < 12){commandLine.push("RIGHT")
+setCommandLine(commandLine)
+setCount(commandLine.length)}}
+
+function addLeft(){
+if(count < 12){commandLine.push("LEFT")
+setCommandLine(commandLine)
+setCount(commandLine.length)}}
+
+function addUp(){
+if (count < 12){commandLine.push("UP")
+setCommandLine(commandLine)
+setCount(commandLine.length)}}
+
+function addDown(){
+if (count < 12){
+commandLine.push("DOWN")
+setCommandLine(commandLine)
+setCount(commandLine.length)}}
+
+function del(){
+commandLine.pop()
+setCommandLine(commandLine)
```

Comment:
Here is an example on how to remove the last item without mutation:

```suggestion
setCommandLine(commandLine.slice(0, -1));
```

Reply:
https://developer.mozilla.org/de/docs/Web/JavaScript/Reference/Global_Objects/Array/slice